src/context.cc

```diff
@@ -359,6 +359,7 @@ UIElements UIElements::Reset() {
     render.display_unsynced_items = true;
 
     render.open_time_entry_list = true;
+    render.display_timeline = true;
```

Comment:
I'd render the timeline by default because the Timeline UI and TimeEntryList are rendering in main app.